wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+# Use this script to test if a given TCP host/port are available
+
+WAITFORIT_cmdname=${0##*/}
```

Comment:
What wizardry is this? 😅

Reply:
The number of repos I've put this in 😆 

As for why, we have to wait for ceramic and composedb to be available.